Cargo.toml

```diff
@@ -14,3 +14,16 @@ pest = "2.1.3"
 pest_derive = "2.1.0"
 proc-macro2 = "1.0.8"
 quote = "1.0.3"
+
+[dev-dependencies]
+trybuild = "1.0.27"
+tempfile = "3.1.0"
+anyhow = "1.0.28"
```

Comment:
Can we sort these? I use `cargo add -s` usually and that'll do that on the next add operation ;)

Reply:
my `cargo add` doesn't support dev dependencies